app/models/vacation.rb

```diff
@@ -28,6 +28,12 @@ class Vacation < ApplicationRecord
     .order(start_date: :asc, end_date: :asc)
   }
 
+  scope :expired, -> {
+    where(status: :pending)
```

Comment:
just to let you know, the enumerize has the scope set so you can just call `.pending` here